pkg/vcs/gitlab_client/client.go

```diff
@@ -175,7 +160,12 @@ func (c *Client) CreateHook(ctx context.Context, repoName, webhookUrl, webhookSe
 	return nil
 }
 
-func buildRepoFromEvent(event *gitlab.MergeEvent) *repo.Repo {
+func (c *Client) LoadHook(ctx context.Context, id string) (*repo.Repo, error) {
+	//TODO implement me
```

Comment:
Should we gate the `process` command logic to only work on `vcs-type == github` since Gitlab has no impl defined?

Reply:
Whoops, forgot to implement that. Got that done, should be good now.